Process.h

```diff
@@ -241,6 +297,18 @@ static inline bool Process_isChildOf(const Process* this, pid_t pid) {
    return pid == Process_getParentPid(this);
 }
 
+#define Process_isKernelThread(_process) ((_process)->isKernelThread)
+#define Process_isUserlandThread(_process) ((_process)->isUserlandThread)
+
+static inline bool Process_isThread(const Process *this) {
+   return Process_isUserlandThread(this) || Process_isKernelThread(this);
+}
+
+#define CMDLINE_HIGHLIGHT_FLAG_SEPARATOR  0x00000001
+#define CMDLINE_HIGHLIGHT_FLAG_BASENAME   0x00000002
+#define CMDLINE_HIGHLIGHT_FLAG_COMM       0x00000004
+#define CMDLINE_HIGHLIGHT_FLAG_DELETED    0x00000008
```

Comment:
These macros seem to be only used in `Process.c`, maybe also define them there?

Reply:
As the MergeCommand structure is public interface, so should the constants used in that publically accessible interface.